src/handle_action.cpp

```diff
@@ -675,14 +702,24 @@ static void wait()
         durations[retval] = duration;
     };
 
-    add_menu_item( 1, '1', !has_watch ? _( "Wait 300 heartbeats" ) : "", MINUTES( 5 ) );
-    add_menu_item( 2, '2', !has_watch ? _( "Wait 1800 heartbeats" ) : "", MINUTES( 30 ) );
+    if( setting_alarm ) {
+
+        add_menu_item( 0, '0', _( "" ), MINUTES( 30 ) );
```

Comment:
Is there a need for translating `""`?

Reply:
Oversight from edits. I briefly had text there during testing and I was being neurotic about strings. Will fix.